packages/apollo-language-server/src/diagnostics.ts

```diff
@@ -72,11 +72,25 @@ export function diagnosticsFromError(
       source: `GraphQL: ${type}`,
       message: error.message,
       severity,
-      range: rangeForASTNode(highlightNodeForNode(node) || node)
```

Comment:
This is the earliest line I can comment on, but should L65 (the return type definition `Diagnostic[]` be updated to `GraphQLDiagnostic[]`?

Reply:
Yes!